packages/shylu/shylu_dd/bddc/src/shylu_SolverFactoryBDDC.h

```diff
@@ -80,6 +78,15 @@
 #include "shylu_SolverClusterPardiso.h"
 #endif
 
+#if defined(HAVE_SHYLU_DDBDDC_MUELU)
+#include "shylu_SolverMueLu.h"
+#endif
+
+//#define NODALAMGBDDC
+#ifdef NODALAMGBDDC
```

Comment:
Is there a way to build without NODALMGBDDC ? I thought this was always built by default ?

Reply:
The capabilities in NodalAMG will be included in a future pull request. Right now the build would not include NodalAMG since NODALAMGBDDC is not defined.